internal/lookout/ui/src/utils/jobsDialogUtils.ts

```diff
@@ -1,12 +1,12 @@
 import _ from "lodash"
-import { JobFilter, Job } from "models/lookoutV2Models"
+import { Job, JobFilter } from "models/lookoutV2Models"
 import { IGetJobsService } from "services/lookoutV2/GetJobsService"
 
 export const getAllJobsMatchingFilters = async (
   filters: JobFilter[],
   getJobsService: IGetJobsService,
 ): Promise<Job[]> => {
-  const MAX_JOBS_PER_REQUEST = 1000
+  const MAX_JOBS_PER_REQUEST = 10000
```

Comment:
it's ok to increase this by a factor of 10x?

Reply:
I believe so - API should be able to serve 10000 jobs at a time. It will result in fewer requests being made for the same requests.

Other option is having this be configurable